plugins/org.brainwy.liclipsetext.editor/plugin.xml

```diff
@@ -216,7 +216,7 @@
     <!-- Note: run the install.py to regenerate the actions contents! -->
     <extension point="org.eclipse.core.contenttype.contentTypes">
         <content-type base-type="org.eclipse.core.runtime.xml"
-            file-extensions="xml,xsd" file-names=".pydevproject,.project,.classpath"
+            file-extensions="xml,xsd" file-names=".pydevproject,.project,.classpath,.launch"
```

Comment:
Hi Paul, thanks for the pull request, still, this is automatically generated (see comment just above your change). You should just change https://github.com/fabioz/LiClipseText/blob/master/plugins/org.brainwy.liclipsetext.editor/install.py and re-run it with python (2.7).